modules/market/src/main/java/com/opengamma/strata/market/curve/CurveCalibrationInfo.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.market.curve;
+
+/**
+ * Information about curve calibration.
+ * <p>
+ * Implementations of this interface provide information that was obtained during curve calibration.
+ */
+public interface CurveCalibrationInfo {
```

Comment:
The implementation is in a different package, so an interface is needed. Plus, it avoids locking in our specific way of doing curve calibration. The convert method gives a little more flexibility to us in the future, such as converting data on the fly between formats.